mappings/net/minecraft/container/CombinationContainer.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_4861 net/minecraft/container/CombinationContainer
+	FIELD field_22479 resultInv Lnet/minecraft/class_1263;
+	FIELD field_22480 inputsInv Lnet/minecraft/class_1263;
```

Comment:
Should this be plural (`inputsInv`) or singular (`inputInv`)?

Reply:
I think `inputInv`/`inputInventory` would be better.

Reply:
@Juuxel I didn't use 'inventory' as the suffix since the other containers didn't either, though that should probably be discussed as I think it is more natural.

Reply:
Note in my PR I simply called them `input` and `output`.